docs/networks/node-ops/access-onchain-data/access-nodes/access-node-configuration-options.md

```diff
@@ -78,21 +78,57 @@ The URL in that file will point to a file named `root.checkpoint`. This is the b
 gsutil -m cp "gs://flow-genesis-bootstrap/[network]-execution/public-root-information/root.checkpoint*" .
 ```
 
+Where `[network]` is the network you are downloading for. For example, `mainnet-24` or `testnet-49`.
+
 Once the files are downloaded, you can either move them to `/bootstrap/execution-state/` within the node’s bootstrap directory or put them in any mounted directory and reference the location with this cli flag: `--execution-state-checkpoint=/path/to/root.checkpoint`. The naming of files should be `root.checkpoint.*`.
 
 ## **Option 2: Enabling Indexing Mid-Spork**
 
-### Download the root protocol state snapshot
+### Identify the root checkpoint
 
-You can download  `root-protocol-state-snapshot.json`  file by using the `GetProtocolStateSnapshotByHeight` call with the corresponding height. You will get a `base64` encoded snapshot which you will need to decode.
+The root checkpoint for the network is used by Execution and Access nodes to bootstrap their local execution state database with a known trusted snapshot. The checkpoint contains 18 files that make up the merkle trie used to store the blockchain’s state.
 
-Store the **`root-protocol-state-snapshot.json`** into the **`/bootstrap/public-root-information/`** folder.
+Root checkpoints are periodically generated on Flow Foundation execution nodes and uploaded to a GCP bucket. You can see 
+a list of available checkpoints [here](https://console.cloud.google.com/storage/browser/flow-genesis-bootstrap/checkpoints), 
+or list them using the `gsutil` command
+
+```bash
+gsutil ls "gs://flow-genesis-bootstrap/checkpoints/"
+```
+
+The checkpoint paths are in the format `flow-genesis-bootstrap/checkpoints/[network]/[epoch number]-[block height]/`.
+Where
+* `[network]` is the network the checkpoint is from. For example, `mainnet` or `testnet`.
+* `[epoch number]` is the epoch number of the checkpoint. You can find the current epoch number on the [flowdiver](https://www.flowdiver.io/) home page. 
+* `[block height]` is the block height of the checkpoint.
+Make sure that the checkpoint you select is from an epoch when your node was part of the network.
```

Comment:
it can be any of epoch in which the node was part of the network, correct?

Reply:
yea